.github/workflows/ci-cd.yaml

```diff
@@ -251,14 +257,14 @@ jobs:
         /scripts/init_osc_creds.sh
         mkdir -p $HOME/.config/osc
         cp /root/.config/osc/oscrc $HOME/.config/osc
-    - name: prepare tranto.changes file
+    - name: Prepare trento.changes file
       run: |
-        VERSION=$(./hack/get_version_from_git.sh)
+        VERSION=${{steps.getvers.outputs.tag}}
```

Comment:
It would loose the integer date and the commit-hex in the suffix.
for example it would be `trento-0.4.1.tar.gz` instead of `trento-0.4.1+dev51.1633446165.75e2eef.tar.gz`

Reply:
yep, nice catch. I thought in the beggining that the relevant thing would have been to have the correct version in the trento binary but I guess this would prevent it from properly updating the RPM after a new rolling RPM is out.